spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -232,6 +232,8 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private volatile int defaultTxSize = DEFAULT_TX_SIZE;
 
+	protected volatile boolean defaultDurableSubscription = true;
+
```

Comment:
I don't **think** I'm being inconsistent - in #357 I'm saying we should log or reject invalid global properties; in this case (if durableSubscription is global), the kafka binder should reject an invalid setting for it. The solution here (for #357) is to demote it instead.

Pushing properties like this down to the implementations removes any possibility of doing such logging and the properties are simply ignored.

I guess I am ok (but not thrilled) with very specific properties being ignored by other binders, but there is definitely a grey area with "somewhat" common properties. One concern is how do we ensure consistency across binders (e.g. someone creates a JMS binder and names the property something else, e.g. `durable`).